src/contracts/core/DelegationManager.sol

```diff
@@ -422,6 +422,68 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         }
     }
 
+    /**
+     * @notice Called by an avs to register an operator with the avs.
+     * @param operator The address of the operator to register.
+     * @param operatorSignature The signature, salt, and expiry of the operator's signature.
+     */
+    function registerOperatorToAVS(
```

Comment:
Shouldn't this method check that `operator` is an operator? IMO this method should continue the flow that `registerAsOperator` starts

Reply:
this is a good point; we should add the check and test coverage which makes sure the function reverts appropriately when called by an address that has never registered as an operator.
there's an `isOperator` method which we use for the same check elsewhere in the DelegationManager, e.g. here: https://github.com/Layr-Labs/eigenlayer-contracts/blob/783797b307ad49b51acac36a6837ed8fc664601b/src/contracts/core/DelegationManager.sol#L120 -- so you can copy-paste that check + just update the revert string.

Reply:
Good idea, makes sense to me

Reply:
done